pkg/pricer/export_test.go

```diff
@@ -0,0 +1,14 @@
+// Copyright 2020 The Swarm Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package pricer
```

Comment:
Would be good if you can add a comment here (https://github.com/ethersphere/bee/issues/1135)